certora/README.md

```diff
@@ -38,6 +48,8 @@ The [`certora/specs`](specs) folder contains the following files:
 - [`Reentrancy.spec`](specs/Reentrancy.spec) checks that Morpho token contracts are reentrancy safe by ensuring that no function is making external call;
 - [`Immutability.spec`](specs/Immutability.spec) checks that Morpho token implementation contract is immutable because it doesn't perform any delegate call other than to the upgrade function;
 - [`ERC20.spec`](specs/ERC20.spec) ensure that the Morpho token is compliant with the [ERC20](https://eips.ethereum.org/EIPS/eip-20) specification;
-
+- [`Delegation.spec`](specs/Delegation.spec) checks the logic for voting power delegation is correct.
 
 The [`certora/confs`](confs) folder contains a configuration file for each corresponding specification file for both the Ethereum and the Optimism version.
+
+The [`certora/Makefile`](Makefile)  is used to track and perform the required modifications on source files.
```

Comment:
```suggestion
The [`certora/Makefile`](Makefile) is used to track and perform the required modifications on source files.
```

Reply:
Fixed in https://github.com/morpho-org/morpho-token-upgradeable/pull/92/commits/f80c7fbaebcbaf0f9b92f71aeafeed59a04c15fc.